src/web/components/Core/SelectAllCheckbox.scss

```diff
@@ -0,0 +1,21 @@
+.select-all-checkbox {
+  background-color: var(--theme-radio-background) !important;
+  width: 21px;
+  height: 21px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  // Override the style of .app button
+  padding: 0 !important;
+  border: 0 !important;
+}
+
+.select-all-checkbox.uncheck {
```

Comment:
I'd like to see this done using some of the available SCSS features - you could nest this inside the previous block, probably using the parent selector: https://sass-lang.com/documentation/style-rules/parent-selector